src/ray/gcs/redis_context.cc

```diff
@@ -122,16 +127,21 @@ Status CallbackReply::ReadAsStatus() const {
   return status_reply_;
 }
 
-std::string CallbackReply::ReadAsString() const {
+const std::string &CallbackReply::ReadAsString() const {
```

Comment:
I'm not too familiar with this code. @micafan can you help confirm if it's safe to return a reference here?

Reply:
It's ok return ref here. The string is parsed and save to the class member, and it's readonly.